scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -996,20 +999,22 @@
                                          e)
                     (throw e)))))
 
-            (handle-launch-task-metrics matches count-txns pool-name compute-cluster)
-
             ;; Launching the matched tasks MUST happen after the above transaction in
             ;; order to allow a transaction failure (due to failed preconditions)
-            ;; to block the launch
+            ;; to block the launch.
             (prom/with-duration
               prom/scheduler-launch-all-matched-tasks-submit-duration {:pool pool-name}
               (timers/time!
                 (timers/timer (metric-title "handle-resource-offer!-mesos-submit-duration" pool-name))
                 (let [_ (log-structured/info "Launching matched tasks for compute cluster"
                                              {:pool pool-name :compute-cluster compute-cluster-name})]
-                  (doseq [match matches]
-                    (timers/stop (-> match :leases first :offer :offer-match-timer))
-                    (-> match :leases first :offer :offer-match-timer-prom-stop-fn))
+                  ;; Offers must be launched within a timeout, otherwise they expire worthless.
```

Comment:
"**Mesos** offers must be launched .... only applicable to **Mesos tasks on Fenzo** ..... "

"When Mesos goes away, this metric ceases to be important and this can be deleted."

Reply:
Done